include/aspect/vof_initial_conditions/interface.h

```diff
@@ -0,0 +1,211 @@
+/*
+  Copyright (C) 2011, 2012 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__vof_initial_conditions_interface_h
+#define __aspect__vof_initial_conditions_interface_h
+
+#include <aspect/plugins.h>
+#include <aspect/geometry_model/interface.h>
+
+#include <deal.II/base/point.h>
+#include <deal.II/base/parameter_handler.h>
+
+namespace aspect
+{
+  /**
+   * A namespace in which we define everything that has to do with defining
+   * the initial conditions.
+   *
+   * @ingroup VoFInitialConditionsModels
+   */
+  namespace VoFInitialConditions
+  {
+    using namespace dealii;
+
+    /**
+     * A structure that contains enum values that identify type of
+     * initialization. Interpolation approach has less resolution than desired,
+     * and the nature of the data allows a more accurate approximation.
+     */
+    struct VoFInitType
+    {
+      enum Kind
+      {
+        /**
+         * Initialization data is a composition value between 0 and 1 at all
+         * points and should be accumulated by integration.
+         */
+        Compos,
+        /**
+         * Initialization data is an interface defined by a signed distance
+         * level set with positive value indicating fluid presence.
+         */
+        SDist_LS
```

Comment:
Same here: `SignedDistanceLeastSquares` or `signed_distance_least_squares` (we tend to name the values of enums in lower case and underscore)